src/config.rs

```diff
@@ -13,54 +13,27 @@ use lazy_static::lazy_static;
 use log::{debug, info, set_max_level, warn};
 use parking_lot::Mutex;
 use sentry::types::Dsn;
-use serde::Deserialize;
 
 use crate::constants::DEFAULT_MAX_DIF_ITEM_SIZE;
 use crate::constants::DEFAULT_MAX_DIF_UPLOAD_SIZE;
 use crate::constants::{CONFIG_RC_FILE_NAME, DEFAULT_RETRIES, DEFAULT_URL};
+use crate::utils::auth_token::AuthToken;
+use crate::utils::auth_token::AuthTokenPayload;
 use crate::utils::http::is_absolute_url;
 
 /// Represents the auth information
 #[derive(Debug, Clone)]
 pub enum Auth {
     Key(String),
+    #[deprecated = "Use Auth::AuthToken instead"]
```

Comment:
Using `#[deprecated]` for completely internal things (we do not publish `sentry-cli` as a crate) is not really useful.

Reply:
> we do not publish `sentry-cli` as a crate

Did not realize that it wasn't published, so I thought this was a public API that we would need to remain backwards compatible with. In this case, I will just remove this enum variant, since it should never be getting created with my changes.